java/apps/authentication-portal/src/main/webapp/authenticate.jsp

```diff
@@ -1,12 +1,12 @@
 <%--
-  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ Copyright (c) 2018-2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
We need to remove the phrase "All Rights Reserved" as well from the license headers in open source code

Reply:
I'll fix in a followup PR since the PR is needed for Alpha.

Reply:
https://github.com/wso2/identity-apps/pull/3935

PR sent. cc: / @pavinduLakshan